west.yml

```diff
@@ -137,7 +137,7 @@ manifest:
       groups:
         - fs
     - name: hal_adi
-      revision: a3eecfde1c76d38312b94fd346c7ba9fe2661992
+      revision: pull/10/head
```

Comment:
zephyrproject-rtos/hal_adi#10 has now been merged, so this can be updated to the new SHA

Reply:
Updated, thank you :)